python/sbp/client/drivers/network_drivers.py

```diff
@@ -88,8 +88,8 @@ def write(self, s):
 class HTTPException(Exception):
   pass
 
-DEFAULT_CONNECT_TIMEOUT = 10
-DEFAULT_READ_TIMEOUT = 10
+DEFAULT_CONNECT_TIMEOUT = 30
+DEFAULT_READ_TIMEOUT = 120
 DEFAULT_TIMEOUT = (DEFAULT_CONNECT_TIMEOUT, DEFAULT_READ_TIMEOUT)
 MAX_CONNECT_RETRIES = 10
```

Comment:
Yeah, that sounds good - 3 or 5?

There will be some downsides to this increased timeout during connection outages, but it will start acting more like a long poll and not be so frenetic on first connections.